tests/onegov/org/test_browser.py

```diff
@@ -439,3 +440,32 @@ def test_external_map_link(browser, client):
     browser.visit('/topics/themen')
     browser.find_by_css(".button-state").click()
     assert browser.is_text_present("Karte Geo-BS")
+
+
+@pytest.mark.flaky(reruns=3)
+def test_context_specific_function_are_displayed_in_person_directory(
```

Comment:
The exact same test with `client` somehow did not work. So that's the reason I used `browser`.